### ISSUE TITLE
BucketAlreadyOwnedByYou error when running `replibyte bucket list` multiple times

### ISSUE THREAD

**Issue by ederski** [Qovery/Replibyte#85]
When running `replibyte bucket list`, I get the following error if the `bridge` bucket already exists.

```
[2022-05-05T20:40:28Z ERROR replibyte::bridge::s3] BucketAlreadyOwnedByYou: Your previous request to create the named bucket succeeded and you already own it.
failed to create bucket 'revere-staging-sync-service-data'
```

**Comment by ederski** [comment]
This also seems to error for every replibyte command I run. I was able to export a backup by first deleting the bucket, but now when I try to run restore from that backup (`replibyte -c ./replibyte.yaml restore local --image postgresql --value latest`), I get the BucketAlreadyOwnedByYou error.

**Comment by wtait1-ff** [comment]
Hey @ederski I followed along with your discussion in the `#replibyte` Discord from yesterday as I noted you're trying to use `replibyte` in GCP, as I'm currently doing the same evaluation 🙂 . So just wanted to clarify for the maintainers that this issue is experienced specifically in scope for Google Cloud Storage as the datastore provider - but just wanted to double-check with you that that's the case?

I also just filed https://github.com/Qovery/replibyte/issues/89 to improve documentation for GCP support.

**Comment by ederski** [comment]
Hi! 👋  Yes I am using GCP and this issue is probably specific to Cloud Storage, totally forgot to mention that 🤦‍♀️ . Thanks for the help, documentation for GCP would be fantastic :)

**Comment by evoxmusic** [comment]
I am going to take a look this weekend 👍🏽

**Comment by evoxmusic** [comment]
It will be fixed with v0.6.0 - which will be released in a few minutes.
